### PR TITLE
Add title to select customer page

### PR DESCRIPTION
**Pull request by pierre-H**

### CODE REVIEW

**Comment by bartoszpietrzak1994** [comment]
Hi Pierre!

Thank you for your contribution. Could you provide a screenshot presenting your changes?

**Comment by pierre-H** [comment]
>  Hi Pierre!
> Thank you for your contribution. Could you provide a screenshot presenting your changes?

image

File: src/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -18,3 +18,4 @@ sylius_admin_order_creation:
         order_preview: 'Order preview'
         reorder: 'Reorder'
         shipping_methods_selection_requirement: 'You need to add some items and shipping address to select from eligible shipping method'
+        customer_selection: 'Customer selection'
```
Comment (Zales0123) [inline src/Resources/translations/messages.en.yml]: Alphabetical order FTW :)

Comment (pierre-H) [reply src/Resources/translations/messages.en.yml]: done

**Comment by Zales0123** [comment]
Thank you, Pierre! :tada:

**Pull request merged**